onnxruntime/core/providers/dml/DmlExecutionProvider/src/TensorDesc.cpp

```diff
@@ -210,13 +210,29 @@
     return { m_strides, m_strides + m_bufferTensorDesc.DimensionCount };
 }
 
+void TensorDesc::SetStrides(gsl::span<const uint32_t> strides)
+{
+    if (!strides.empty())
+    {
+        // TODO: Verify strides size does not overwrite memory of m_strides.
```

Comment:
>// TODO: Verify strides size does not overwrite memory of m_strides. [](http://example.com/codeflow?start=8&length=68)

What is the maximum size of m_strides, and what happens if someone passes a larger strides than that?